projects/centerformer/README.md

```diff
@@ -0,0 +1,99 @@
+# CenterFormer: Center-based Transformer for 3D Object Detection
```

Comment:
I recommend to rename this project to `CenterFormer`, the path should be `projects/CenterFormer/centerformer/...`